_data/banking.yml

```
@@ -1444,32 +1808,42 @@ websites:
       - phone
       - hardware
     doc: https://www.wellsfargo.com/privacy-security/advanced-access
+    regions:
+      - us
 
   - name: Wescom Credit Union
     url: https://www.wescom.org/
     twitter: Liz_Wescom
```

```suggestion
    twitter: _wescom
```
Updated Twitter handle.